fix: add redirect for the old scopes page

david-hamilton-glean opened the pull request:
Glean's admin -> Client API Tokens -> OAuth card has a "learn more" link that goes to `/docs/client_api/client_api_scopes`.  Add a redirect from that to the Glean-issued tokens scopes section.

rwjblue-glean commented:
> this comment likely belongs in the OpenAPI repo, but it's worth noting that these code examples don't include instance in their examples.

Yep!! That's exactly one of the things I'm trying to fix (over in gleanwork/open-api). Step one is making a new "code sample post processor" which is why I needed to refactor this ingestion pipeline to use a stable directory.

rwjblue-glean commented:
FWIW, #61 will fix the link-rot issue.

rwjblue-glean commented:
Validated that all of the new links work, and that the local build functions properly.

steve-calvert-glean commented:
@rwjblue-glean can you help me understand the rationale for renaming the final output? The names before we clear and unambiguous that they were both the spec _and_ the code samples. If this is part of a larger restructuring of the `gleanwork/open-api` repo that makes sense, but it's hard to grok how this change relates.

steve-calvert-glean commented:
Sure, but is that important? Does it matter? It's actually useful to be able to infer the eventual state of things by just reading the file. But again, I don't have that strong of an opinion, but I reserve the right to change it back if it doesn't "feel" right :D

steve-calvert-glean commented:
I'm inclined to just merge this change as is, because as I said it's very low on the priority list of what I'm concerned about. Thanks for doing this!

david-hamilton-glean committed:
fix: add redirect for the old scopes page

Glean's admin -> Client API Tokens -> OAuth card has a "learn more" link
that goes to `/docs/client_api/client_api_scopes`.  Add a redirect from
that to the Glean-issued tokens scopes section.
